packages/matrix/tests/realm-urls.spec.ts

```diff
@@ -33,7 +31,7 @@ test.describe('Realm URLs in Matrix account data', () => {
       '@user1:localhost',
       user.accessToken,
       'com.cardstack.boxel.realms',
-      JSON.stringify({ realms: [testRealmURL] }),
+      JSON.stringify({ realms: [] }),
```

Comment:
this test was actually wrong based on the current implementation of the realm server. Currently public realms need to opt out of being catalog realms--this includes our test realms which are considered to be catalog realms since we have not opted out of this behavior. As such we should not include the test realm as user assigned realm since that mechanism is only for private and shared realms.

Reply:
@FadhlanR please confirm that public realms need to opt out of being catalog realms